onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -199,6 +199,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "(2, batch_size, num_heads, max_sequence_length, head_size)",
               AttributeProto::INT,
               static_cast<int64_t>(0))
+        .Attr("mask_filter_value",
```

Comment:
which parity test can best show the parity change W/o this attribute?

Reply:
python/tools/transformers/models/gpt2/convert_to_onnx.py can be used to test parity as well. It will report parity with pytorch.

You can also run onnxruntime/test/python/transformers/test_parity_huggingface_gpt_attention.py.

I think you will also need update some test models (expected output models of fusion if it has Attention.) and update document as well.